src/pages/form-validation/table/table-validation.test.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
```

Comment:
why eslint-disable here?

Reply:
I think this was a copy/paste gone wrong.  This file was already linted corrected & the comment was removed.